content/blog/2016/2016-10-27-xunit-reporting.adoc

```diff
@@ -0,0 +1,225 @@
+---
+layout: post
+title: "xUnit and Pipeline"
+tags:
+- pipeline
+- plugins
+- xunit
+- nightwatch
+author: lnewman
+---
+
+NOTE: This is a guest post by link:https://github.com/bitwiseman[Liam Newman],
+Technical Evangelist at link:http://cloudbees.com[CloudBees].
+
+The
+link:https://wiki.jenkins-ci.org/display/JENKINS/JUnit+Plugin[JUnit plugin]
+is the go-to test result reporter for many Jenkins projects,
+but the it is not the only one available.  The
+link:https://wiki.jenkins-ci.org/display/JENKINS/xUnit+Plugin[xUnit plugin]
+is a viable alternative that supports JUnit and many other test result file formats.
+
+=== Introduction
+
+No matter the project, you need to gather and report test results.
```

Comment:
I steadfastly agree but I find plenty of people who don't know about the test reporting integration in Jenkins. I think it would be nice to spice up this blog post with some pretty test result trend graphs towards the top of the blog post

I think you could also include a "Test Failures" screenshot which shows a Jenkins build page with tests which made the build unstable.